docs/integration/cucumberjvm.adoc

```diff
@@ -2,12 +2,17 @@
 
 == Installation
 
-The available latest version of `allure-cucumber-jvm`:
+Each major version of Cucumber JVM requires particular version of Allure Cucumber JVM adapter.
 
-image::https://img.shields.io/maven-central/v/io.qameta.allure/allure-cucumber-jvm.svg[Allure CucumberJVM]
+The available latest version of adapters:
+
+- Cucumber JVM 1.x - `allure-cucumber-jvm` image:https://img.shields.io/maven-central/v/io.qameta.allure/allure-cucumber-jvm.svg[Allure Cucumber JVM 1.x]
+- Cucumber JVM 2.x - `allure-cucumber2-jvm` image:https://img.shields.io/maven-central/v/io.qameta.allure/allure-cucumber2-jvm.svg[Allure Cucumber JVM 2.x]
+- Cucumber JVM 3.x - `allure-cucumber3-jvm` image:https://img.shields.io/maven-central/v/io.qameta.allure/allure-cucumber3-jvm.svg[Allure Cucumber JVM 3.x]
+- Cucumber JVM 4.x - `allure-cucumber4-jvm` image:https://img.shields.io/maven-central/v/io.qameta.allure/allure-cucumber4-jvm.svg[Allure Cucumber JVM 4.x]
 
 === Maven
-Simply add `allure-cucumber-jvm` plugin as dependency to your project and add it to CucumberOptions:
+Simply add `allure-cucumber4-jvm` plugin as dependency to your project and add it to CucumberOptions:
```

Comment:
Simply add `allure-cucumber4-jvm` plugin as **a** dependency to your project and add it to CucumberOptions:

Reply:
Thanks!